### ISSUE TITLE
About DQN some

### ISSUE THREAD

**Issue by guest-oo** [AI4Finance-Foundation/ElegantRL#461]
Dear Author, I see that in your dqn training, eposide, calculation loss and neural network update are not considered according to the original traditional DQN algorithm, and they are updated after a certain number of steps. Do you have the introduction of the paper and the pseudo-code introduction?

**Comment by mozhongzhou** [comment]
自动回复：已收到您的邮件
